src/mongocrypt-opts.c

```diff
@@ -58,6 +65,7 @@ _mongocrypt_opts_cleanup (_mongocrypt_opts_t *opts)
    _mongocrypt_buffer_cleanup (&opts->schema_map);
    _mongocrypt_opts_kms_provider_azure_cleanup (&opts->kms_provider_azure);
    _mongocrypt_opts_kms_provider_gcp_cleanup (&opts->kms_provider_gcp);
+   _mongocrypt_opts_kms_provider_kmip_cleanup (&opts->kms_provider_kmip);
```

Comment:
Ah-hah, one of many, but not releasing/destroying/freeing/cleaning up the entire structure. (Sweep? Mop..? ;-D)

Reply:
Haha, to make it more precise we can consider:
`_mongocrypt_opts_kms_free_fields_but_do_not_free_the_struct` :)